### PR TITLE
test sdk-preview for amlarc compute

### PR DESCRIPTION
**Pull request by zetiaatgithub**
# PR into Azure/azureml-examples

## Checklist

I have:

- [ ] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: .github/amlarc/amlarc-tool.sh [hunk 1]
```diff
@@ -2,8 +2,9 @@
 set -x
 
 # Global variables
-export LOCK_FILE=$0.lock
-export RESULT_FILE=amlarc-test-result.txt
+export SCRIPT_DIR=$( cd  "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+export LOCK_FILE=${SCRIPT_DIR}/$0.lock
+export RESULT_FILE=${SCRIPT_DIR}/amlarc-test-result.txt
 export MAX_RETRIES=60
 export SLEEP_SECONDS=20
```
Comment (snowei) [inline .github/amlarc/amlarc-tool.sh:10]: as we're moving it, I would suggest that we rename it to kubernetes-tool.sh, at least the folder name? @Bozhong68 @Zhong-J what do your think?

File: .github/workflows/cli-scripts-deploy-safe-rollout-k8s-online-endpoints.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ on:
       - cli/deploy-safe-rollout-k8s-online-endpoints.sh
       - .github/workflows/cli-scripts-deploy-safe-rollout-k8s-online-endpoints.yml
       - cli/setup.sh
-      - .github/amlarc-tool.sh
+      #- .github/amlarc/amlarc-tool.sh
```
Comment (snowei) [inline .github/workflows/cli-scripts-deploy-safe-rollout-k8s-online-endpoints.yml:14]: why comment out this?

File: .github/amlarc/amlarc-tool.sh (hunk 1 above)
Comment (Zhong-J) [reply .github/amlarc/amlarc-tool.sh:10]: Agree. "amlarc" wording should be replaced by "kubernetes". After GA, we will not expose amlarc to customers.

File: .github/workflows/cli-scripts-deploy-safe-rollout-k8s-online-endpoints.yml (hunk 2 above)
Comment (zetiaatgithub) [reply .github/workflows/cli-scripts-deploy-safe-rollout-k8s-online-endpoints.yml]: fixed by bringing it back

File: .github/kubernetes-compute/tool.sh [hunk 3]
```diff
@@ -2,8 +2,9 @@
 set -x
 
 # Global variables
-export LOCK_FILE=$0.lock
-export RESULT_FILE=amlarc-test-result.txt
+export SCRIPT_DIR=$( cd  "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+export LOCK_FILE=${SCRIPT_DIR}/$0.lock
+export RESULT_FILE=${SCRIPT_DIR}/amlarc-test-result.txt
```
Comment (snowei) [inline .github/kubernetes-compute/tool.sh:7]: do we need to rename the result file name?

**Review by Zhong-J — APPROVED** [review]

File: .github/workflows/kubernetes-compute-cpu-resources-setup.yml [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      MIN_COUNT: 5
+      CLUSTER_TYPE: managedClusters
+      RELEASE_TRAIN: ${{ github.event.inputs.AMLARC_RELEASE_TRAIN }}
+      KEY_VAULT_NAME: amlarcgithubworkflowkv
+      KEY_VAULT_SUB: dd94e4df-b58f-4e81-9c5e-bd72a4b2ef9e
+      KEY_VAULT_RG: amlarc-github-workflow
+      REINSTALL_EXTENSION: ${{ github.event.inputs.REINSTALL_EXTENSION }}
+      SETUP_DATASET: ${{ github.event.inputs.SETUP_DATASET }}
+
+      SEVERITY: 4
```
Comment (snowei) [inline .github/workflows/kubernetes-compute-cpu-resources-setup.yml:45]: don't change it to 3?

File: .github/workflows/kubernetes-compute-gpu-resources-setup.yml [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      VM_SKU: STANDARD_NC12
+      CLUSTER_TYPE: managedClusters
+      RELEASE_TRAIN: ${{ github.event.inputs.AMLARC_RELEASE_TRAIN }}
+      KEY_VAULT_NAME: amlarcgithubworkflowkv
+      KEY_VAULT_SUB: dd94e4df-b58f-4e81-9c5e-bd72a4b2ef9e
+      KEY_VAULT_RG: amlarc-github-workflow
+      REINSTALL_EXTENSION: ${{ github.event.inputs.REINSTALL_EXTENSION }}
+      SETUP_DATASET: ${{ github.event.inputs.SETUP_DATASET }}
+
+      SEVERITY: 4
```
Comment (snowei) [inline .github/workflows/kubernetes-compute-gpu-resources-setup.yml:44]: also here.

File: .github/workflows/kubernetes-compute-cpu-resources-setup.yml (hunk 4 above)
Comment (zetiaatgithub) [reply .github/workflows/kubernetes-compute-cpu-resources-setup.yml:45]: changed

File: .github/workflows/kubernetes-compute-gpu-resources-setup.yml (hunk 5 above)
Comment (zetiaatgithub) [reply .github/workflows/kubernetes-compute-gpu-resources-setup.yml:44]: changed

**Review by snowei — COMMENTED** [review]
IcM sev level

File: .github/kubernetes-compute/tool.sh (hunk 3 above)
Comment (zetiaatgithub) [reply .github/kubernetes-compute/tool.sh:7]: changed

**Review by snowei — APPROVED** [review]

**Pull request merged**